ecc/Makefile

```diff
@@ -72,9 +78,9 @@ docker-run:
 		chaincode
 
 docker-stop:
-	-docker stop $(DOCKER_CONTAINER)
-	-docker rm $(DOCKER_CONTAINER)
+	-if [ ! -z "$(DOCKER_CONTAINER_ID)" ]; then docker stop $(DOCKER_CONTAINER); fi
```

Comment:
this should be `docker stop $(DOCKER_CONTAINER_ID);`, right?

also DOCKER_CONTAINER is not really useful anymore, I think DOCKER_CONTAINER_ID is much better.

I suggest let's remove DOCKER_CONTAINER and in docker-run target we just use CC_NAME.

Reply:
agreed